src/main/resources/application.yaml

```diff
@@ -1,5 +1,18 @@
 path.content.root: ./content/
 write.access: false
 
+management:
+  endpoint.health.show-details: always
+  endpoints.web:
+    exposure.include: info, health, metrics, prometheus
+    base-path: "/actuator"
```

Comment:
I also thought about using /internal, but then every app use dokgen needs to change the naiserator, otherwise they are not able to deploy, because of the livness etc. endpoints changed.
I am just not sure who/which app are affected by such change so keep /actuator for safety

Reply:
Why? We use internal for all our apps. Did we use actuator before converting to kotlin?

Reply:
Yes, I think it was. But not in Dokgen because it did not explicitly expose these endpoints (but /status and /info were exposed by default). Because the default base url is "/actuator", familie-ba-dokgen assumeed the url, so did familie-ef-maler:
```
spec:
  image: {{ image }}
  port: 8080
  liveness:
    path: /actuator/info
    initialDelay: 30
    timeout: 5
    periodSeconds: 10
    failureThreshold: 27
  readiness:
    path: /actuator/health
    initialDelay: 30
    timeout: 5
    periodSeconds: 10
    failureThreshold: 27
```

Reply:
hmm, ok 👍  I dont like it, but I see your point :)

Reply:
Either we keep dokgen to use /actuator, or we change familie-ba-dokgen to use /internal.

Reply:
We can use actuator 👍